test/test_tabel.py

```diff
@@ -27,10 +27,11 @@ def tbls():
             Tabel(),
             Tabel([1,2,3]),
             Tabel({'a':range(10),'b':range(10)}),
+            Tabel({'a':range(10),'b':list(map(str, range(10)))}),
             Tabel([range(10),range(10),range(10)]),
             Tabel(np.array([np.arange(10),np.arange(10,20)])),
             Tabel({'a':12,'b':13}),
-            Tabel(pd.DataFrame({'a':[12]*6, 'b':np.arange(6), 'c':list(map(str, range(6)))})),
```

Comment:
Why do you want to delete this tabel? Ideally we keep supporting all tables we ever decided to support,...